lib/cdo/google/drive.rb

```diff
@@ -54,6 +54,15 @@ def file(path)
       return nil
     end
 
+    def file_by_id(id)
+      file = @session.file_by_id(id)
+      return nil if file.nil?
+      Google::Drive::File.new(@session, file)
+    rescue GoogleDrive::Error => exception
+      CDO.log.info "Error accessing Google Drive file with id: #{id} because: #{exception.message}"
```

Comment:
nit: we should be consistent within this file, and either use `ChatClient.log` here, or use `CDO.log.info` in the `file` method above

Reply:
I missed that one. I've completely removed called to `ChatClient` and also have updated the error handling logic. Previously, we were swallowing errors and logging them to Slack. We should raise an error if a file can't be downloaded or if it's modified time can't be accessed.